Source/WebInspectorUI/UserInterface/Base/Main.js

```diff
@@ -50,6 +50,9 @@ WI.LayoutDirection = {
 
 WI.loaded = function()
 {
+    // Ports may or may not include additions.
+    window.WebKitAdditions ??= {};
```

Comment:
hmm kinda odd that this is inside `WI.loaded`

`WebKitAdditions/WebInspectorUI/WebInspectorUIAdditions.js` is already included at the end of `Main.html`, so I think it'd probably be easier to just have that file completely override whatever was previously declared instead of having this do a `??=`

as such, why dont we move this to be a `var WebKitAdditions = {};` inside `WebInspector.js` (i.e. alongside where `WI` is first declared)?